### DISCUSSION TITLE
امکان جستجو ip در cdn های دیگر

### DISCUSSION THREAD

**Discussion by MiladMk** [MortezaBashsiz/CFScanner#251] [General]
با سلام خواستم بدونم این نرم افزار امکان جستجو در cdn های دیگه رو هم داره یا خیر و اگه داره چطور ؟
چون من مثلا رنج ip آروان کلود رو به برنامه دادم و کانیگی که کار می کنه رو هم دادم ولی برنامه چیزی رو پیدا نمی کنه.
حتی ip تکی که کار می کنه رو دادم به برنامه بازم برنامه میزنه not working.
در ضمن TLS HandShake هم تست کردم فعال بود.

**Comment by xmohammad1** [comment]
دقیقا منم میخواستم رنج ای پی های اروان رو اسکن کنم ببینم کدوم بهتر کار میکنه روی کانفیگم ولی نمیشد انگار

**Comment by xmohammad1** [comment]
اگه میشه راهنمایی کنید چطوری میشه رنج ای پی cdn های دیگه مثل اروان رو اسکن کرد
